webui/src/stories/storyData/README.md

```diff
@@ -0,0 +1,11 @@
+This directory is for creating fake data for mocks.
+
+You need to install bun (we're using bun because it's simple and outside of our project toolkit):
```

Comment:
Is bun a hard requirement?  There are a lot of tools already, and introducing another JS variant seems unnecessary?

Reply:
removed